source/documentation/ontology/__index.md

```diff
@@ -10,6 +10,9 @@ won't go into all of the many details of the API here: you should
 expect to refer to the [Javadoc](/documentation/javadoc/jena/) to
 get full details of the capabilities of the API.
 
+_Please note that this section covers the new Jena ontology API, which has been introduced since Jena 5.1.0.
+The legacy Jena Ontology API documentation see [here](legacy.html)._
```

Comment:
Maybe s/The legacy/For the legacy

Or /API documentation see [here/API documentation can be found [here.../

Reply:
fixed